src/config.js

```diff
@@ -0,0 +1,31 @@
+class Config {
+    constructor() {
+        this.default = {
```

Comment:
Can't think of a better name for this but it seems funny that we're overriding something called `default`

Reply:
Wouldn't something like `items` be better, or `values`.

Reply:
Good point, I'll change it to values tomorrow